subsys/bluetooth/mesh/rpl.c

```diff
@@ -200,7 +200,9 @@ static struct bt_mesh_rpl *bt_mesh_rpl_alloc(uint16_t src)
 
 void bt_mesh_rpl_reset(void)
 {
-	int i;
+	uint32_t shift = 0;
+	uint32_t last = 0;
+	uint32_t i;
```

Comment:
Can i be decleared as int i = 0 in the for loop?

Reply:
Instead of having it in the top it, could have been decleared in the for (int i = 0; ...